docs/bugs-features.rst

```diff
@@ -0,0 +1,155 @@
+|             | - ``Refactor`` Improvement that will not be visible to the user in any way.       |
+|             | - ``Task`` Infrastructure work that will not be a part of released code.          |
++-------------+-----------------------------------------------------------------------------------+
+| Subject     | - For an ``Issue``, summary of the situation and the unexpected result.           |
+|             | - For a ``Story``, takes the form "As a [user/dev/etc] I can ..."                 |
+|             | - For a ``Task`` or ``Refactor`` describe what should be done. in any way.        |
++-------------+-----------------------------------------------------------------------------------+
+| Description | A detailed explanation of the problem please see :ref:`issue-description`         |
++-------------+-----------------------------------------------------------------------------------+
+| Status      | - ``NEW`` Unassigned, incomplete                                                  |
+|             | - ``ASSIGNED`` Incomplete, assignee should also be set                            |
+|             | - ``POST`` Pull Request is open (with a link in a comment)                        |
+|             | - ``MODIFIED`` Change has been merged, but has not been released                  |
+|             | - ``CLOSED`` If you disagree, please re-open and comment                          |
++-------------+-----------------------------------------------------------------------------------+
+| Priority    | Assigned during :ref:`triage`.                                                    |
++-------------+-----------------------------------------------------------------------------------+
+| Assignee    | Contributor who is working on this issue.                                         |
++-------------+-----------------------------------------------------------------------------------+
+| Milestone   | A set of work that has been grouped together.                                     |
++-------------+-----------------------------------------------------------------------------------+
+| Parent Task | Indicates that this is a sub-task of the larger issue.                            |
++-------------+-----------------------------------------------------------------------------------+
+| Severity    | Assigned during :ref:`triage`.                                                    |
++-------------+-----------------------------------------------------------------------------------+
+| Version     | Filer experienced the problem while running this version of pulpcore              |
++-------------+-----------------------------------------------------------------------------------+
+| Platform    | - This field is set only on issues that have been completed                       |
+| Release     | - indicates the earliest version that contains these  changes                     |
```

Comment:
Capitalize "indicates" and consider swapping the order of these two bullet points